airflow/providers/google/cloud/operators/datapipeline.py

```diff
@@ -81,27 +81,21 @@ class RunDataPipelineOperator(GoogleCloudBaseOperator):
     """ Run Data Pipeline Operator """
     def __init__(
             self,
-            data_pipeline_name: str = "{{task.task_id}}",
-            project_id: str | None = None,
-            location: str = DEFAULT_DATAPIPELINE_LOCATION,
+            response_body: dict,
```

Comment:
Take in only the pipeline name as a parameter, as other data in the response_body will be unused. When running a pipeline created by the create operator, extract the name from the created pipeline in the DAG itself.

Reply:
Instructed to use only the pipeline name as input. Closing out this request and updating